src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2931,4 +2931,12 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="UpdateAvailable.Title" xml:space="preserve">
     <value>Update available</value>
   </data>
+  <data name="FileExplorerPreview_Toggle_Monaco_Max_File_Size.Header" xml:space="preserve">
+    <value>Maximum file size to preview</value>
```

Comment:
We should mention the unit. Maybe (if possible) we should convert it in code to KB to have a smaller value.

Reply:
I mention it in the description

Reply:
Another point of view: this sentence and the first of the description are _almost_ the same. Maybe it's good to have one good sentence as the header and one sentence for the description.